### ISSUE TITLE
.net standard support

### ISSUE THREAD

**Issue by yongwang** [nielscup/ShareFile#5]
Since .NET standard 2.0 have been release. Is there any plan to add support for .NET standard?
Thanks
Yong

**Comment by nielscup** [comment]
Definitely, I will be updating some of my own apps to .net standard soon and update this plugin accordingly.

**Comment by 5yunus2efendi** [comment]
Can you use net .NET Standard 1.0? I look into the code and I think it's enough to use only .NET Standard 1.0
Thanks
